doc/changelog.rst

```diff
@@ -100,6 +100,11 @@ Breaking Changes in 4.0
 - Removed :class:`bson.binary.UUIDLegacy`.
 - The "tls" install extra is no longer necessary or supported and will be
   ignored by pip.
+- ``directConnection`` defaults to False instead of None, allowing for the
```

Comment:
I would say:
```
- The ``directConnection`` URI option and keyword argument to
  :class:`~pymongo.mongo_client.MongoClient` defaults to ``False`` instead of ``None``...
```

Reply:
Done.